cpp/ps0000/p0000-soln.cpp

```diff
@@ -1,9 +1,7 @@
-#include <iostream> 
-
-using namespace std;
-
-int main(void) 
+#include <iostream> // 2/2
+int main(void) // 1/1
 {
-  cout << "Hello, World!\n"; 
-  return 0;
+  std::cout << "Hello, World!\n";  // 4/4
+  return 0; // 1/1
 }
+// great work 😎 10/10 ⭐

```

Comment:
- [x] all code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit